XML/TestCases/FunctionalTests.xml

```diff
@@ -785,4 +785,20 @@
         <Tags>cpu</Tags>
         <Priority>3</Priority>
     </test>
+    <test>
+        <testName>HANDLE-OFFLINED-CPU-TO-VMBUS-CHANNEL</testName>
+        <testScript>CPU-Offline.ps1</testScript>
+        <setupType>OneVM</setupType>
+        <OverrideVMSize>Standard_L32s_v2</OverrideVMSize>
+        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\CPUOfflineKernelBuild.sh,.\Testscripts\Linux\handle_offline_cpu.sh</files>
+        <TestParameters>
+            <param>repo_url=CPU_CustomKernelURL</param>
```

Comment:
I recently observed that we are following all caps convention for all Replaceable parameters.
Can you fix this?

Reply:
CPU_CustomKernelURL > CPU_CUSTOMKERNELURL Like this?

Reply:
yes

Reply:
Fixed in 42ab2b700a561179b910110c0d370addeb5e77ab

Reply:
Can you check once again for "HANDLE-OFFLINED-CPU-TO-VMBUS-CHANNEL" parameters? I can still see older convention

Reply:
Good catch. I did not save the other 2 files. Can you see right ones?